molecule/default/molecule.yml

```diff
@@ -9,7 +9,14 @@ lint: |
     flake8
 platforms:
   - name: instance
-    image: centos:7
+    image: rockylinux:9
+    image_version: latest
```

Comment:
Not sure all these extra options are necessary as they mostly apply for the systemd base image.

Reply:
Thanks, will fix in #6 - needed to rename this branch